hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -335,6 +335,24 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "ENCODING.SPARSE",
+      desc: "Sparse manufacturing state encoding."
+    }
+    { name: "MAIN.FSM.SPARSE",
+      desc: "The main state FSM is sparsely encoded."
+    }
+    { name: "KMAC.FSM.SPARSE",
+      desc: "The KMAC interface FSM is sparsely encoded."
+    }
+    { name: "MAIN.FSM.ESCALATION",
```

Comment:
Just to make sure I understand, this is different from `LOCAL_ESC` right?

Reply:
Indeed, I've added this to distinguish it from global escalation via the life cycle signals. Maybe we should align the names to something like `GLOBAL_ESC` / `LOCAL_ESC` to make the distinction clear?

Reply:
Thanks for changing that, `GLOBAL_ESC` sounds perfect to me. It would probably make sense to have a public document somewhere explaining the general difference (as well as all other assets and terms).